controllers/replica_set_controller.go

```diff
@@ -50,9 +51,6 @@ import (
 const (
 	clusterDNSName = "CLUSTER_DNS_NAME"
 
-	// lastVersionAnnotationKey should indicate which version of MongoDB was last
-	// configured
-	lastVersionAnnotationKey    = "mongodb.com/v1.lastVersion"
```

Comment:
This was actually not used, moved it and renamed (`lastVersion` would have been misleading in enterprise)